<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditTree</class>
 <widget class="QWidget" name="FormEditTree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>451</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gestion des arbres</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2" rowspan="3">
    <widget class="QListView" name="listTypesView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>161</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonExpandAll">
     <property name="text">
      <string>Déplier tout</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonCollapseAll">
     <property name="text">
      <string>Replier tout</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QToolBox" name="toolBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tree">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>774</width>
        <height>96</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Arbres</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="newTreeButton">
         <property name="text">
          <string>Ajouter &amp;arbre</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="duplicateTreeButton">
         <property name="text">
          <string>&amp;Dupliquer arbre</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="importTreeButton">
         <property name="text">
          <string>&amp;Importer arbre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="deleteTreeButton">
         <property name="text">
          <string>&amp;Supprimer arbre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="finishTreeButton">
         <property name="text">
          <string>&amp;Terminer arbre</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="randomTreeButton">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Arbre aléatoire</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QPushButton" name="viewTreeButton">
         <property name="text">
          <string>Statistiques, impression et exportation</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nodes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>774</width>
        <height>68</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Noeuds</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="addNodeButton">
         <property name="text">
          <string>&amp;Ajouter noeud</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="deleteNodeButton">
         <property name="text">
          <string>S&amp;upprimer noeud</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="addTypeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ajouter type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="deleteTypeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Supprimer type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QPushButton" name="modifyNodeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&amp;Modifier noeud</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>listTypesView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
